mariadb/content.md

```diff
@@ -133,6 +139,12 @@ Do note that there is no need to use this mechanism to create the root superuser
 
 By default, the entrypoint script automatically loads the timezone data needed for the `CONVERT_TZ()` function. If it is not needed, any non-empty value disables timezone loading.
 
+### `MARIADB_AUTO_UPGRADE` / `MARIADB_DISABLE_UPGRADE_BACKUP`
```

Comment:
Just thinking about it: Since this might indeed change the `mysql` database of an pre-existing datadir, shall we add a note about this here too?